FinSearchUnified/Tests/Bundle/SearchBundleFindologic/QueryBuilderTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace FinSearchUnified\Tests\Bundle\SearchBundleFindologic;
+
+use Exception;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder;
+use Shopware\Components\HttpClient\GuzzleHttpClient;
+use Shopware\Components\HttpClient\RequestException;
+use Shopware\Components\HttpClient\Response;
+use Shopware\Components\Test\Plugin\TestCase;
+
+class QueryBuilderTest extends TestCase
+{
+    /**
+     * Data provider for testing scenarios of findologic request execution
+     *
+     * @return array
+     */
+    public function querybuilderResponseProvider()
```

Comment:
* Change the method name to `aliveResponseProvider`.
* For the data set descriptions: Uppercase the first letter so it reads `Response` instead of `response`.